core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -99,18 +102,23 @@ public static synchronized TachyonFS get(final TachyonURI tachyonURI) throws IOE
    * @param masterHost master host details
    * @param masterPort port master listens on
    * @param zookeeperMode use zookeeper
+   * @param tachyonConf The TachyonConf instance.
    * 
-   * @return the corresponding TachyonFS hanlder
+   * @return the corresponding TachyonFS handler
    * @throws IOException
    */
-  public static synchronized TachyonFS get(String masterHost, int masterPort, boolean zookeeperMode)
-      throws IOException {
-    return new TachyonFS(new InetSocketAddress(masterHost, masterPort), zookeeperMode);
+  public static synchronized TachyonFS get(String masterHost, int masterPort,
+      boolean zookeeperMode, TachyonConf tachyonConf) throws IOException {
+    if (tachyonConf != null) {
+      tachyonConf.set(Constants.USE_ZOOKEEPER, Boolean.toString(zookeeperMode));
+      tachyonConf.set(Constants.MASTER_HOSTNAME, masterHost);
+      tachyonConf.set(Constants.MASTER_PORT, Integer.toString(masterPort));
+    }
```

Comment:
Do we set these explicitly because they are not required in the client configuration, I thought set was more of a tool for testing purposes?

Also, is it okay if tachyonConf is null? I think we directly try to access the zookeeper mode later.